src/Symfony/Component/AssetMapper/AssetMapperRepository.php

```diff
@@ -76,7 +76,7 @@ public function findLogicalPath(string $filesystemPath): ?string
         }
 
         foreach ($this->getDirectories() as $path => $namespace) {
-            if (!str_starts_with($filesystemPath, $path)) {
+            if (!str_starts_with($filesystemPath, $path.'/')) {
```

Comment:
Be careful. Import map paths might already have a trailing `/`: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap#mapping_path_prefixes

Reply:
I think we're ok. These are all comparing filesystem paths - not the actual paths that are exposed to the user / importmap.